app/controllers/properties_controller.rb

```diff
@@ -36,11 +36,24 @@ def pricing
     @properties = Property.find(params[:id]) 
   end
 
+  def description
```

Comment:
1. For `def create`, instead of using `@properties`, use `@property`
2. Separate the current_user from property like this
    ```
@property = Property.new(property_params)
@property.user_id = current_user.id
```
Since you don't need the USER id yet
![Screen Shot 2021-05-20 at 8 09 06 PM](https://user-images.githubusercontent.com/35232348/118976148-48622280-b9a7-11eb-8ccf-08ea0fccaebb.png)

Reply:
After separating the two IDs, you can now save data
![Screen Shot 2021-05-20 at 8 13 43 PM](https://user-images.githubusercontent.com/35232348/118976821-0be2f680-b9a8-11eb-894a-6d84e053d3cb.png)

HINT for next step would be:
1. To remove the NILL values and replace them with real data, you will use UPDATE
2. Use if @property.update(@property_params), follow what you did for CATEGORY in Elearning